src/com/skillsup/view/LoginMenu.java

```diff
@@ -26,85 +28,165 @@ public void show() {
+                } else {
+                    newUser = new User(newLogin, newPassword);
+                }
+                Response<User> registerResponse = userService.register(newUser, choiceUserDB);
+                System.out.println(registerResponse.getFinalMessage());
+            } else System.out.println("Rejection of changes is accepted!");
+        } else {
+            System.out.println(loginResponse.getFinalMessage());
         }
-        return gender;
     }
 
-
-    private void loginSubMenu() {
-        System.out.println("your login:");
+    private void deleteUserSubMenu(String choiceUserDB) {
+        System.out.println("Input your login:");
         String login = scanner.nextLine();
+        System.out.println("Input your password:");
+        String password = scanner.nextLine();
+        Response<User> loginResponse = userService.login(login, password, choiceUserDB);
+        if (loginResponse.isSuccess()) {
+            User user = loginResponse.getData();
+            System.out.println("User logged in :" + user);
+            String procceding = chooseYesOrNoMenu("Would You like to delete this user? (y/n): ", scanner);
+            if ("y".equalsIgnoreCase(procceding)) {
+                System.out.println(userService.delete(user, choiceUserDB).getFinalMessage());
+            } else System.out.println("Rejection of deleting is accepted!");
+        } else {
+            System.out.println(loginResponse.getFinalMessage());
+        }
+    }
 
-        System.out.println("your password:");
+    private void changeAccountBalanceSubMenu(String choiceUserDB) {
+        System.out.println("Input your login:");
+        String login = scanner.nextLine();
+        System.out.println("Input your password:");
         String password = scanner.nextLine();
+        Response<User> loginResponse = userService.login(login, password, choiceUserDB);
+        if (loginResponse.isSuccess()) {
+            User user = loginResponse.getData();
+            System.out.println("Current user info: " + user.toStringTotal());
+            String procceding = chooseYesOrNoMenu("Would You like to change account balance? (y/n): ", scanner);
+            if ("y".equalsIgnoreCase(procceding)) {
+                userService.delete(user, choiceUserDB);
+                user.setAccount(BigDecimal.valueOf(getDoubleInput("Input your new account balance: ", scanner)));
+                Response<User> changeAccountResponse = userService.changeAccountBalance(user, choiceUserDB);
+                System.out.println(changeAccountResponse.getFinalMessage());
+            } else System.out.println("Rejection of changes is accepted!");
+        } else {
+            System.out.println(loginResponse.getFinalMessage());
+        }
+    }
 
-        Response<User> loginResponse = userService.login(login, password);
+    private void getAllUsersSubMenu(String choiceUserDB) {
+        System.out.println("Input your login:");
```

Comment:
login/pass can be reused, used "session" approach (remembering credentials somewhere in static object) 
we dont want user to login again on each operation